frontend/src/app/(auth)/signin/page.tsx

```diff
@@ -37,7 +32,13 @@ export default function SignInPage() {
 
   // TODO: only temp until authentication and roles is fully working
   if (mutation.isSuccess) {
-    console.log(mutation.data.data);
+    signInContext.setAuth({
```

Comment:
saving token and user email so I can fetch candidate profile data on the candidate profile page